ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -2240,20 +2240,22 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-mtu-migrate
-- as: gcp-ipi-p3-f28-long-duration-part3-3
+- as: gcp-ipi-filestore-csi-p3-f28-long-duration-part3-3
```

Comment:
@chao007 
1, I am not sure if you change the workflow, does it impact Cluster_Operator|OTA?
2, how many time does it increase? as I known, there is big buffer for Cluster_Operator|OTA which already take long time to need to add more cases. here is the information of duration https://github.com/openshift/release/pull/39873#issuecomment-1573208799 when OTA add to this test.

cc @jianping-shu @shellyyang1989 to be aware of this.

so, my suggestion is that you'd better add your own new test with profile you want, like `aws-ipi-p2-f28-long-duration-apiserver`, mabye your name is `gcp-ipi-filestore-csi-p3-f28-long-duration-storage`

Reply:
@kuiwang02 We have 3 test cases and need 0.5 hours.

Reply:
@chao007 what is the duration in the future?

for me, I do not suggest it. 
but it depends on cluster_operator and ota.
you could query it from them.

Reply:
The extra 0.5 hour is fine with me. It's up to Jianping.

Reply:
> @chao007 what is the duration in the future?
> 
> for me, I do not suggest it. but it depends on cluster_operator and ota. you could query it from them.

Will not have much test cases.  
We only add longduration test cases when used `enterprise` type filestore volume for special feature. Most of time, we use `standard` type of filestore volume.

Reply:
@jianping-shu how about you?

Reply:
@chao007 Thanks for the input! I think the increased time is ok for this gcp long duration job.
Will you do the similar thing for aws long-duration-part3-3 job? 
The most part of Hive auto cases will be on aws i.e. in aws long-duration-part3-3 job, we may need to split the job in the future since the 23h limit will not be enough. So this aws job is better to be dedicated for Hive only. Just want you know this background and make better planning on ci jobs.

Reply:
@jianping-shu I only update the gcp long-duration-part3-3 job, will not update on the aws platform. Thanks for the kind of remind.

Reply:
Thanks! LGTM